src/Nuget/Microsoft.VisualStudio.ProjectSystem.Managed.VS/Microsoft.VisualStudio.ProjectSystem.Managed.VS.Nuspec

```diff
@@ -0,0 +1,38 @@
+<?xml version="1.0"?>
+<package xmlns="http://schemas.microsoft.com/packaging/2011/08/nuspec.xsd">
+  <metadata>
+    <id>Microsoft.VisualStudio.ProjectSystem.Managed.VS</id>
+    <summary>"Microsoft VisualStudio Managed Project System VS Components</summary>
+    <description>
+      Microsoft VisualStudio ProjectSystem for C#/VB(Managed) Projects which interact with VisualStudio interfaces
```

Comment:
Microsoft VisualStudio ProjectSystem for C#/VB(Managed) Projects for hosts that interact with VisualStudio interfaces.